Exercise_4/submit.bash

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#SBATCH --partition=gpu		#Set partition to "gpu"
+#SBATCH -J exercise2		#Set job name
+#SBATCH --time=00:10:00		#Set job time limit
+#SBATCH --nodes=1		#Set number of nodes to 1
+#SBATCH --ntasks=1		#Set number of cpus to 1
+#SBATCH --gres=gpu:1		#Request 1 GPU
+#SBATCH --mem=10Gb		#Request 10GB of RAM memory
+#SBATCH --output=%A-%a.out      #set output filename with main job ID and task array ID
+#SBATCH --error=%A-%a.err       #set error filename with main job ID and task array ID
+#SBATCH --array=1-10%1		#execute 10 array jobs, 1 at a time.
```

Comment:
Based on this current setup, it's not obvious why array job is required here. I have the same doubt about Exercise_2 as well.